### DISCUSSION TITLE
[Script request] Alta Labs Self-Hosted Controller

### DISCUSSION THREAD

**Discussion by Azulsky** [community-scripts/ProxmoxVE#439] [Request script]
### Application Name

Alta Labs Control

### Website

https://www.alta.inc/control

### Description

Alta Labs is a Prosumer / SMB Networking Line of hardware for routing, switching, and Wifi.
Their controller can be hosted on their own cloud, using a separate hardware appliance, or locally!

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
What do you need for this? Do you need special hardware? We can't just blindly put in scripts and hope that it works.

**Reply by Azulsky** [reply]
> What do you need for this? Do you need special hardware? We can't just blindly put in scripts and hope that it works.

This install is equivalent to the unifi controller for another vendor that specializes in prosumer networking.

They just started offering the controller as a virtual appliance so i thought it would be good to add.
